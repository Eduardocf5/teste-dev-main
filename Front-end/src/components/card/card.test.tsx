import renderer from 'react-test-renderer';
import Card from '.';

const value = {
    id: 1,
    name: "Pica pau",
    img: "https://img.elo7.com.br/product/zoom/3FFF8A3/matriz-de-bordado-pica-pau-matriz-picapau.jpg",
    status: "Ave",
    species: "Picidae"
}

test('render test', () => {
    const component = renderer.create(<Card
        id={value.id}
        name={value.name}
        img={value.img}
        status={value.status}
        species={value.species}
        />).toJSON();

    expect(component).toMatchSnapshot();
  });