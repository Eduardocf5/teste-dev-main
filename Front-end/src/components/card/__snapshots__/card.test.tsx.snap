// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render test 1`] = `
<div
  className="bg-white flex items-center cursor-pointer hover:shadow-inner p-4 rounded-md gap-9"
>
  <img
    alt="icon"
    className="rounded-full w-24"
    src="https://img.elo7.com.br/product/zoom/3FFF8A3/matriz-de-bordado-pica-pau-matriz-picapau.jpg"
  />
  <div>
    <div
      className="text-lg"
    >
      Pica pau
    </div>
    <div
      className="flex items-center gap-1"
    >
      <div
        className="rounded-full undefined w-3 h-3"
      />
      <div>
        Ave
         - 
        Picidae
      </div>
    </div>
  </div>
</div>
`;
